streams/cy.m3u

```diff
@@ -1,10 +1,6 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="AdaTV.cy",ADA TV (576p)
 https://sc-kuzeykibrissmarttv.ercdn.net/adatv/bant1/playlist.m3u8
-#EXTINF:-1 tvg-id="AlfaSport.cy",Alfa Sport (1080p) [Not 24/7]
-https://dev.aftermind.xyz/edge-hls/unitrust/alfasports/index.m3u8?token=8TXWzhY3h6jrzqEqu
```

Comment:
Make sure to fix all of the rest channels where 8TXWzhY3h6jrzqEqu -> 8TXWzhY3h6jrzqEqx